lefthook.yaml

```diff
@@ -2,4 +2,4 @@ pre-commit:
   commands:
     check:
       glob: "*.{js,ts,json}"
-      run: biome check --apply {staged_files} && git add {staged_files}
```

Comment:
Do you know if this change was intended? It doesn't seem to be part of the PR

Reply:
In my case, I cloned this repo for the first time, and I haven't had a globally installed biome before. Because of that, this validation failed when I tried to commit.

This change uses the local biome version installed with pnpm instead of the global version, which is not guaranteed to be installed on the machine.

Reply:
Yeah, that makes sense!